From: Remi Collet <remi@remirepo.net>
Date: Tue, 7 Jun 2022 09:57:15 +0200
Subject: NEWS

(cherry picked from commit f451082baf14ee9ea86cdd19870e906adb368f02)
(cherry picked from commit 87247fb08e905e629836350ac4e639edd1b40ed8)
---
 NEWS | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/NEWS b/NEWS
index a7ed5a7..59ed4dc 100644
--- a/NEWS
+++ b/NEWS
@@ -1,6 +1,16 @@
 PHP                                                                        NEWS
 |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
 
+Backported from 7.4.30
+
+- mysqlnd:
+  . Fixed bug #81719: mysqlnd/pdo password buffer overflow.
+    (CVE-2022-31626) (c dot fol at ambionics dot io)
+
+- pgsql
+  . Fixed bug #81720: Uninitialized array in pg_query_params().
+    (CVE-2022-31625) (cmb)
+
 Backported from 7.3.33
 
 - XML:
